static/app/components/tabs/tab.tsx

```diff
@@ -23,6 +23,8 @@ interface TabProps extends AriaTabProps {
    */
   overflowing: boolean;
   state: TabListState<any>;
+  hideSelection?: boolean;
+  showPressed?: boolean;
```

Comment:
What does `showPressed` mean? Not sure that this prop makes sense. We might want to think of a more descriptive way to modify the styling here.